### ISSUE TITLE
Django/uwsgi - timeouts after period of time

### ISSUE THREAD

**Issue by lgrabowski** [unbit/uwsgi#735]
Hello, 
it may be related to uwsgi or not - I do not know - issue which I am facing is after like 10/13 h of providing service - uwsgi dies. Process are up - but no response in written over the socket where nginx is listening which leads us to t/o. 
-system is doesn't semd any signals (application is not killed due to e.g huge memory usage)
-we are not suffering on lack of inodes/filedescriptors or HDD space

We cant reload uwsgi - its stucks after 
"has been touched... grace them all !!! ***
...gracefully killing workers...
Gracefully killing worker 1 (pid: 12950)...
Gracefully killing worker 4 (pid: 12953)...
Gracefully killing worker 3 (pid: 12952)...
Gracefully killing worker 2 (pid: 12951)..."

we need to send sigkill to uwsgi to kill it and run again.

our configuration is:
Django 1.6 
uWSGI 2.0.6 (64bit) on [Fri Oct  3 16:00:35 2014] compiled with version: 4.6.3
Linux-3.5.0-54-generic #81~precise1-Ubuntu SMP Tue Jul 15 04:02:22 UTC 2014
nginx with directive:
uwsgi_pass unix:///some/path/uwsgi.sock;

uwgi setup:
[uwsgi]
chdir=some_path
module=some_module.wsgi:application
master=True
pidfile=some_path
vacuum=True
max-requests=5000
reload-mercy=5
processes=4
home=some_path/
pythonpath=some_many_other_paths
socket=some_path/uwsgi.sock
daemonize=some/path/some_log.log
buffer-size=30000
uid=some_user
gid=some_user
touch-reload=some/path

// given config is with path palceholders

changing max-requests - only leads us to ealier or later problem (depends on give value)

Stability time depends on amount of requests - when I perform some stress tests - uwsgi goes in to this "zombie" state much faster then on regular user traffic.
to sum up - after some period of time I am unable to reload uwsgi and it's not responding.
